src/test/java/EventGetterTest.java

```diff
@@ -0,0 +1,202 @@
+import data_gateway.event.CalendarManager;
+import data_gateway.event.EventReader;
+import entity.Event;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import services.event_creation.EventInfoFromReader;
+import services.event_presentation.CalendarEventPresenter;
+import services.event_presentation.EventGetter;
+import services.event_presentation.EventInfo;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.*;
```

Comment:
Change your IDE settings to not bundle imports into the wildcard (you should be able to set the count to like 999)

Reply:
fixed